extensions/gdx-setup/src/com/badlogic/gdx/setup/Dependency.java

```diff
@@ -37,4 +37,13 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
good point, and totally true, I just thought that because in case of hash code I have to write more code to generate the integer and this is the only use case possible in entire app it would be a quick solution. Because if I would go for every detail, I would have to use other unique identifiers for extension instead of extension name. And I didn't want to over complicate  it.